src/vs/workbench/services/configurationResolver/common/variableResolver.ts

```diff
@@ -225,7 +225,11 @@ export class AbstractVariableResolverService implements IConfigurationResolverSe
 						return normalizeDriveLetter(getFolderUri().fsPath);
 
 					case 'cwd':
-						return (folderUri ? normalizeDriveLetter(getFolderUri().fsPath) : process.cwd());
+						try {
```

Comment:
Instead of try catch, let's check if there's an `argument`.

Reply:
Done.